scripts/ci

```diff
@@ -325,9 +328,22 @@ def dpkg_binary(dsc_path, name, git, series, build_arch, build_all):
     build_log = path.join(BINARY_DIR, build_log_filename)
     failure_build_log = path.join(FAILURES_DIR, build_log_filename)
 
+    retry_keys = [
+        name,
+        'source:' + source_name,
+        'git:' + git.id,
+        'dist:' + series.codename,
+        'arch:' + build_arch,
+    ]
+
+    retry = False
+    for retry_key in retry_keys:
+        if args.retry.contains(retry_key):
+            retry = True
```

Comment:
I believe this is equivalent to `retry = not set(retry_keys).isdisjoint(args.retry)`, for what it's worth.

Reply:
I was sure there was a more pythony way but I don't think that is readable

Reply:
Yeah, debatable. Using sets is probably faster, but I doubt it matters.